packages/synapse-react-client/src/components/CookiesNotification/CookiesNotification.tsx

```diff
@@ -1,42 +1,107 @@
-import React from 'react'
+import React, { useState } from 'react'
 import FullWidthAlert from '../FullWidthAlert'
+import { Link, Typography } from '@mui/material'
+import CookiePreferencesDialog from './CookiePreferencesDialog'
+import {
+  COOKIES_AGREEMENT_LOCALSTORAGE_KEY,
+  CookiePreference,
+  useCookiePreferences,
+} from '../../utils/hooks/useCookiePreferences'
 
 export const alertConfig = {
   title: 'Our site uses cookies.',
-  description:
-    'This website uses cookies to enhance your experience and to analyze our traffic. Using this website means that you agree with our cookie policy.',
-  primaryButtonText: 'ACCEPT AND CONTINUE',
+  description: (
+    <Typography variant="body1">
+      We use necessary cookies and store your data to ensure our websites
+      function properly. With your consent, we would also like to use cookies to
+      remember your preferences and improve our websites. Review our{' '}
+      <Link href="https://s3.amazonaws.com/static.synapse.org/governance/SynapsePrivacyPolicy.pdf">
+        Privacy Policy
+      </Link>
+      . By clicking “Accept All” you consent to our use of cookies. You can
+      change your cookie preference at any time from the Settings page.
+    </Typography>
+  ),
+  primaryButtonText: 'ALLOW ALL',
+  secondaryButtonText: 'Disable All',
+  tertiaryButtonText: 'Customize',
 }
 
-export const COOKIES_AGREEMENT_LOCALSTORAGE_KEY =
-  'org.sagebionetworks.security.cookies.portal.notification.okclicked'
+const allowAll: CookiePreference = {
+  functionalAllowed: true,
+  analyticsAllowed: true,
+}
+const allowNone: CookiePreference = {
+  functionalAllowed: false,
+  analyticsAllowed: false,
+}
```

Comment:
You also have `allowNone` created in useCookiePreferences.ts. I suggest moving these constant objects there and reuse / import them here